sklearn/linear_model/_bayes.py

```diff
@@ -394,12 +412,12 @@ def _update_coef_(
                 [X.T, U / (eigen_vals_ + lambda_ / alpha_)[None, :], U.T, y]
             )
 
-        rmse_ = np.sum((y - np.dot(X, coef_)) ** 2)
+        mse_ = np.sum((y - np.dot(X, coef_)) ** 2)
```

Comment:
Since [this](https://github.com/scikit-learn/scikit-learn/pull/30644/files#r1918058199) is not trivial, let's add an inline comment:


```suggestion
        # Note: we do not need to explicit use the weights in this sum because
        # y and X were proprocessed by _rescale_data to handle the weights.
        mse_ = np.sum((y - np.dot(X, coef_)) ** 2)
```

Reply:
Also note: `mse_` is still not an accurate name, since we use `np.sum` instead of `np.mean`.

Shall we rename to `sse` instead? Or even `weighted_sse`?

Reply:
Ah true, good catch !